spec/qos-classifier/02-integration_spec.lua

```diff
@@ -0,0 +1,181 @@
+local helpers = require "spec.helpers"
+local cjson = require "cjson"
+
+local PLUGIN_NAME = "qos-classifier"
+
+local config = {
+  termination = { 
+      status_code = 302,
+      header_name= "Location",
+      header_value="https://cred.club"
+  },
+  upstream_header_name = "X-QOS-CLASS",
+  node_count = {
+      http_timeout_in_ms = 15,
+      update_initial_delay_in_sec = 5,
+      initial = 1,
+      update_frequency_in_sec = 1
+  }
+}
+
+local function make_class(green, orange, red)
```

Comment:
nit: can be renamed to `set_classes_with_thresholds` to make it more intuitive

Reply:
Done